charts/opensrp-web/values.yaml

```diff
@@ -145,10 +146,10 @@ reactEnvironmentVariables:
   REACT_APP_OPENSRP_AUTHORIZATION_URL: "https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/auth"
   REACT_APP_OPENSRP_CLIENT_ID: "{{ .Values.sharedVars.opensrpClientId }}"
   REACT_APP_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
-  REACT_APP_OPENSRP_OAUTH_SCOPES: "profile"
+  REACT_APP_OPENSRP_OAUTH_SCOPES: "openid,profile"
   REACT_APP_OPENSRP_OAUTH_STATE: "opensrp"
-  REACT_APP_OPENSRP_ROLES: '{"USERS": "ROLE_EDIT_KEYCLOAK_USERS,ROLE_VIEW_KEYCLOAK_USERS","PLANS": "ROLE_VIEW_KEYCLOAK_USERS","LOCATIONS": "ROLE_VIEW_KEYCLOAK_USERS","CARD_SUPPORT": "ROLE_VIEW_KEYCLOAK_USERS","INVENTORY": "ROLE_VIEW_KEYCLOAK_USERS","TEAMS": "ROLE_VIEW_KEYCLOAK_USERS","PRODUCT_CATALOGUE": "ROLE_VIEW_KEYCLOAK_USERS","FORM_CONFIGURATION": "ROLE_VIEW_KEYCLOAK_USERS","CARE_TEAM": "ROLE_VIEW_KEYCLOAK_USERS","SERVER_SETTINGS": "ROLE_VIEW_KEYCLOAK_USERS", "QUEST": "ROLE_VIEW_KEYCLOAK_USERS"}'
-  REACT_APP_OPENSRP_USER_URL: "https://{{ .Values.sharedVars.opensrpServer }}/opensrp/user-details/"
+  REACT_APP_OPENSRP_ROLES: '{"USERS": "ROLE_EDIT_KEYCLOAK_USERS,ROLE_VIEW_KEYCLOAK_USERS","PLANS": "ROLE_VIEW_KEYCLOAK_USERS","LOCATIONS": "ROLE_VIEW_KEYCLOAK_USERS","CARD_SUPPORT": "ROLE_VIEW_KEYCLOAK_USERS","INVENTORY": "ROLE_VIEW_KEYCLOAK_USERS","TEAMS": "ROLE_VIEW_KEYCLOAK_USERS","PRODUCT_CATALOGUE": "ROLE_VIEW_KEYCLOAK_USERS","FORM_CONFIGURATION": "ROLE_VIEW_KEYCLOAK_USERS","CARE_TEAM": "ROLE_VIEW_KEYCLOAK_USERS","SERVER_SETTINGS": "ROLE_VIEW_KEYCLOAK_USERS", "QUEST": "ROLE_VIEW_KEYCLOAK_USERS", "COMMODITY": "ROLE_VIEW_KEYCLOAK_USERS"}'
+  REACT_APP_OPENSRP_USER_URL: "{{ .Values.sharedVars.userDetailsUrl }}"
```

Comment:
For some reason this does not resolve correctly. This is what is generated in the server:

`REACT_APP_OPENSRP_USER_URL:"https://<no value>/auth/realms/<no value>/protocol/openid-connect/userinfo"`